js/app.js

```diff
@@ -0,0 +1,44 @@
+ const a = 4;
+
+ const b = 53;
+ 
+  const c = 57;
+  
+  const d = 16;
+
+  const e = 'Kevin';
+
+  a < b;
+c > d;
+'Name' _=_ 'Name';
```

Comment:
this is the assignment operator.  the idea was something like this: 
`console.log('Name'==='Name')`